shared/globals.ts

```diff
@@ -27,9 +27,13 @@ export interface Globals extends CommonGlobals {
 	stripeKeyAustralia?: StripePublicKeySet;
 	stripeKeyDefaultCurrencies?: StripePublicKeySet;
 }
+interface QuantumMetricAPIPartial {
+	sendEvent: (param1: number, param2: number, event: string) => void;
```

Comment:
is it this? 
https://github.com/quantummetric/analytics-module-qm/blob/3e927c8c670b2c1f13ae543a9e1863e8afa641bd/src/api/QuantumMetric.ts#L40-L44

Reply:
That looks like the one yep ... it looks like support-frontend do something similar: https://github.com/guardian/support-frontend/blob/a5f4a1b3c8a3c32d14bbeade9601087f1e43c2ea/support-frontend/window.d.ts#L77

I have refactored the interface to make it clear what the functions arguments are.